### PR TITLE
poboljšanje teksta za subscriptione i alertse

### PR DESCRIPTION
**Pull request by tomislavkunaj**
Nisam izmjenio sadržaj bitno, malo izražavanje poboljšao, barem mislim da jesam.

### CODE REVIEW

File: get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Subscriptions and Alerts
 ---
-This article explains how to use Subscriptions and Alerts to schedule SPDocKit reports and send them to the desired email addresses or upload to File share location and SharePoint document library. In order to use this feature, you have to configure an SPDocKit database and the subscription settings, as well as the other delivery methods you prefer.
+This article explains how to use Subscriptions and Alerts to schedule sending SPDocKit reports the either a desired email addresses, or for upload to a file share location or SharePoint document library. In order to use this feature, you have to configure an SPDocKit database and the subscription settings, as well as select the delivery methods you prefer.
```
Comment (vinkobedek) [inline get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md]: i assume you meant to either a

File: get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-Schedule report subscriptions and receive them immediately after the service snapshot, daily, weekly, monthly or quarterly. SPDocKit Service will send them periodically in a configured time. SPDocKit Alerts will be sent immediately after the snapshot if critical data has been detected in Best Practices reports or selected differences report have new information.
+Schedule report subscriptions and receive them on daily, weekly, monthly or quarterly schedule, or even immediately after a snapshot was made by the service. SPDocKit Service will send them periodically at the configured time. SPDocKit Alerts will be sent immediately after the snapshot if critical data has been detected in the Best Practices reports or if the selected differences report has new information.
```
Comment (vinkobedek) [inline get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md]: SPDocKit alerts have a different meaning then what they had when this text was originally written. We should update theis text a bit more to reflect these changes.

File: get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md (hunk 1 above)
Comment (tomislavkunaj) [inline get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md]: je, u pravu si

File: get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md (hunk 2 above)
Comment (tomislavkunaj) [inline get-to-know-spdockit/backstage-screen/subscriptions-and-alerts.md]: Changed the text a bit to try and make it clearer about which version of PDocKit Alerts we're talking about.

**Pull request merged**